doc/design_paper.md

```diff
@@ -1,183 +1,215 @@
 
-Please join our work at tokenscript.org - when it is set-up of course. A Yellow Paper to guide implementors to use Tokenscript for their tokens and dapps will take months to make. However, we keep the process open. Participate now to avoid the draft language specification being made without consideration your token model.
+Previous efforts in this industry primarily focused on enriching the capacity of the technology. This paper will focus on tokenisation and introduce a standardisation effort we call Tokenscript (Token Behaviour Markup Language), which will make the blockchain technical stack complete and provide utility for the economy and the internet. Tokenscript will unleash the full potential of Tokenisation. 
 
-## Abstract
+Please join our work at tokenscript.org. A Yellow Paper to guide implementors to use Tokenscript for their tokens and dapps will take months to make, but a work in progress is always available online. Participate now to avoid the draft language specification being made without considerating your token model.
 
-We recognise the blockchain technology's utility in providing a frictionless market and integrating the web. This is done through tokenization. Tokenized rights can be traded on the market and integrated across systems, forming a frictionless market and allowing free integration.
+\pagebreak
 
+# Abstract
 
-Today, the way tokens are accessed, rendered and transacted are scattered across Dapps and Smart Contracts. If all knowledge about a token is in a dapp, the dapp has to participate in the marketization and all integrations of that token, recreating data interoperability, security and availability barrier - exactly the same set of issues that prevented tokenisation before blockchain was invented.
+We recognise the blockchain technology's utility in providing a frictionless market and integrating the web. This is done through tokenization. Tokenized rights can be traded on the market and integrated across systems, forming a frictionless market and allowing free and accountless integration.
 
-Tokenscript is a method to abstract out the token information, access methods and UI rendering so that they can be efficiently marketized and used for integration.
+Today, the ways tokens are accessed, rendered and transacted are scattered across immutable Smart Contracts on the blockchain. Most token either have a very primitive business logic (as payment token), while some try to put complex business logic into the contract, creating an dapp (decentralized application). This enables DAOs (Decentralized Autonomous Organizations), but hardly helps to create a frictionless token economy and integrate the web. When marketization and integration of a token is tied to a dapp, it recreates data interoperability, security and availability barrier - exactly the same issues that prevented tokenisation before blockchain was invented
```

Comment:
-> scattered across immutable Smart Contracts and Dapps.

The token 𝑟𝑒𝑛𝑑𝑒𝑟𝑖𝑛𝑔 logic is entirely on Dapps. Smart contracts don't have them.